matplotcheck/base.py

```diff
@@ -183,19 +197,29 @@ def assert_caption_contains(self, strings_exp):
         """Asserts that Axes ax contains strings as expected in strings_exp.
         strings_exp is a list of lists. Each internal list is a list of
         strings where at least one string must be in the caption, barring
-        capitalization. Once a string is found, it is removed from the
-        caption, therefore, order does matter. This is to enforce no overlap
-        in found strings.
+        capitalization. For example, ``assert_caption_contains([['A'], ['B'], ['C']])``
```

Comment:
oh~ i didn't realize this was already built into this method! awesome. We will want to write tests that test that both work!!

Reply:
I will address this in another PR and roll this into issue #106

Reply:
excellent!